ext/pdo/tests/bug_78227.phpt

```diff
@@ -0,0 +1,73 @@
+--TEST--
+Bug #78227 Prepared statements ignore RENAME COLUMN
+--EXTENSIONS--
+pdo
+--SKIPIF--
+<?php
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+require_once $dir . 'pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+if (getenv('REDIR_TEST_DIR') === false) putenv('REDIR_TEST_DIR='.__DIR__ . '/../../pdo/tests/');
+require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
+$db = PDOTest::factory();
+@$db->exec("drop table test");
```

Comment:
Why is there a `@`? Please don't silence mysterious warnings. If your intention was to ignore the error when the table doesn't exist then use the SQL syntax `IF EXISTS`

Reply:
`IF EXISTS` is non-standard to my knowledge, and as such might not be supported by all drivers.

Reply:
Thanks for your comment.
This is my mistake, I am so sorry.
My pr is not perfect enough and I will solve it as soon as possible.

Reply:
I am not sure ```IF EXIST```  was supported by all database.